lib/livebook/intellisense.ex

```diff
@@ -519,14 +541,29 @@ defmodule Livebook.Intellisense do
     """
   end
 
-  defp format_docs_link(module, function_or_type \\ nil) do
-    app = Application.get_application(module)
+  defp format_definition_link(module, ctx, function_or_type \\ nil) do
+    if ctx.intellisense_context.ebin_path do
+      path = Path.join(ctx.intellisense_context.ebin_path, "#{module}.beam")
+
+      identifier =
+        if function_or_type,
+          do: function_or_type,
+          else: {:module, module}
 
-    module_name =
-      case Atom.to_string(module) do
-        "Elixir." <> name -> name
-        name -> name
+      with true <- File.exists?(path),
+           {:ok, line} <- Docs.locate_definition(to_charlist(path), identifier) do
+        file = :erpc.call(ctx.node, fn -> module.module_info(:compile)[:source] end)
```

Comment:
Nit: no need for erpc call and node, we can remove the node change from the tests also :)

Reply:
In the Intellisense test for `get_details/4`, the conditions to locate the definition works, but the test fails because the module doesn't exists :/

```
  1) test intellisense completion for remote nodes get details (Livebook.IntellisenseTest)
     test/livebook/intellisense_test.exs:1959
     ** (UndefinedFunctionError) function RemoteModule.module_info/1 is undefined (module RemoteModule is not available)
     code: assert %{contents: [content]} = Intellisense.get_details("RemoteModule", 6, context, node)
     stacktrace:
       RemoteModule.module_info(:compile)
       (livebook 0.15.0-dev) lib/livebook/intellisense.ex:553: Livebook.Intellisense.format_definition_link/3
       (livebook 0.15.0-dev) lib/livebook/intellisense.ex:455: Livebook.Intellisense.format_details_item/2
       (elixir 1.17.2) lib/enum.ex:1703: Enum."-map/2-lists^map/1-1-"/2
       (livebook 0.15.0-dev) lib/livebook/intellisense.ex:420: Livebook.Intellisense.get_details/4
       test/livebook/intellisense_test.exs:1962: (test)
```

Reply:
Oh, I can remove the ebin path, so the module file wouldn't be reachable.